acousticbrainz/views/data.py

```diff
@@ -58,6 +60,13 @@ def get_high_level(mbid):
 def summary(mbid):
     lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
 
+    # Getting good metadata from MusicBrainz
+    try:
+        metadata = musicbrainzngs.get_recording_by_id(
+            mbid, includes=['artists', 'releases', 'media'])['recording']
+    except ResponseError:
+        metadata = None
+
     # Tomahawk player stuff
     if not ('artist' in lowlevel['metadata']['tags'] and 'title' in lowlevel['metadata']['tags']):
```

Comment:
That would be 5 requests for content that changes all the time. Do we really want to do it there?